src/librustc_codegen_llvm/back/lto.rs

```diff
@@ -439,6 +499,8 @@ fn thin_lto(diag_handler: &Handler,
             write::llvm_err(&diag_handler, "failed to prepare thin LTO context".to_string())
         })?;
 
+        let import_map = ThinLTOImports::from_thin_lto_data(data);
```

Comment:
Could the timeline be used to insert statements around this? Calling a callback may take awhile here but it'd be good to benchmark to know!

(in the sense that it's good to optimize this as it'll happen on all incremental release builds unconditionally)

Reply:
👍